README.md

```diff
@@ -64,7 +71,10 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+function doSomething ( thingToDo ) {
+  thingToDo();
+}
+doSomething(thingToDo);
 ```
```

Comment:
I think you have the idea, only thingToDo is not currently defined as anything yet.